src/System.Net.Primitives/src/System/Net/CookieException.cs

```diff
@@ -23,7 +25,6 @@ internal CookieException(string message, Exception inner) : base(message, inner)
         protected CookieException(SerializationInfo serializationInfo, StreamingContext streamingContext)
             : base(serializationInfo, streamingContext)
         {
-            throw new PlatformNotSupportedException();
         }
 
         void ISerializable.GetObjectData(SerializationInfo serializationInfo, StreamingContext streamingContext)
```

Comment:
This explicit impementation is unnecessary, but fine.

Reply:
yah. it's the same as in netfx and considered as a public API. Removing it would be a breaking change I guess...

Reply:
@terrajobst is removing an explicit interface implementation, leaving implicit implementation, a breaking change? I can't think how?

(But it's fine to leave it here)

Reply:
No moving from explicit to implicit interface implementation is not considered a breaking change.